### PR TITLE
Fixing Item Form and implementing NewItemInfo for adding new product

### PR DESCRIPTION
**Pull request by edelveiss**
Form for input info for a new product is fixed. Review page displays newItemInfo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "antd": "^4.4.3",
     "axios": "^0.19.2",
     "craco-less": "^1.17.0",
+    "dotenv": "^8.2.0",
```
Comment (lgv-0) [inline package.json]: Does not have to be installed, react-scripts [installs it as a dependency](https://github.com/Lambda-School-Labs/merchant-marketplace-a-fe/blob/main/package-lock.json#L18300)

Comment (edelveiss) [reply package.json]: Thank you! dotenv is removed from package.json.

**Review by Dustnclay — APPROVED** [review]
great work Tatiana! everything looks good

**Review by aaronmatson1 — APPROVED** [review]
Excellent work!

**Comment by edelveiss** [comment]
[Create NewItemInfo page for adding main info of a new product.](https://trello.com/c/k5LFiKtc/52-create-newiteminfo-page-for-adding-main-info-of-a-new-product)

**Pull request merged**